vmm/sandbox/src/container/handler/spec.rs

```diff
@@ -46,6 +55,7 @@ where
         &self,
         sandbox: &mut KuasarSandbox<T>,
     ) -> containerd_sandbox::error::Result<()> {
+        let shared_path = format!("{}/{}", sandbox.base_dir, SHARED_DIR_SUFFIX);
```

Comment:
should we replace it with get_sandbox_shared_path?

Reply:
Good